tools/remark-preset-lint-node/index.js

```diff
@@ -0,0 +1,44 @@
+// @see https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md
+
+'use strict';
+
+module.exports.plugins = [
+  require('remark-lint'),
+  require('remark-lint-checkbox-content-indent'),
+  require('remark-lint-definition-spacing'),
+  require('remark-lint-fenced-code-flag'),
+  require('remark-lint-final-definition'),
+  require('remark-lint-final-newline'),
+  require('remark-lint-hard-break-spaces'),
```

Comment:
I thought it was possible but `apply.whitespace=fix` strips trailing white spaces when running `git apply` or `git am`
IMHO https://stackoverflow.com/questions/39752867/git-ignore-trailing-whitespace-in-markdown-files-only is a valid solution
```patch
 .gitattributes | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitattributes b/.gitattributes
index fa5162684b..ad2efae71b 100644
--- a/.gitattributes
+++ b/.gitattributes
@@ -1,2 +1,3 @@
 test/fixtures/* -text
 vcbuild.bat text eol=crlf
+**/*.md  whitespace=-trailing-space,space-before-tab,-indent-with-non-tab,tab-in-indent,cr-at-eol
```